src/content/plugins/module-concatenation-plugin.md

```diff
@@ -10,12 +10,13 @@ related:
 
 In the past, one of webpack’s trade-offs when bundling was that each module in your bundle would be wrapped in individual function closures. These wrapper functions made it slower for your JavaScript to execute in the browser. In comparison, tools like Closure Compiler and RollupJS ‘hoist’ or concatenate the scope of all your modules into one closure and allow for your code to have a faster execution time in the browser.
 
-This plugin will enable the same concatenation behavior in webpack.
+This plugin will enable the same concatenation behavior in webpack. It is [enabled by default in production](/configuration/optimization/#optimization-concatenatemodules) but you can also add it manually if you need it in other modes:
```

Comment:
The thing is, linked page already says that this optimization is enabled by default in `production` mode. 
```
Depends on `optimization.providedExports` and `optimization.usedExports`. By default `optimization.concatenateModules` is enabled in production mode and disabled elsewise.
```

Also if you go to mode page: https://webpack.js.org/concepts/mode/#usage is shows you which plugins are enabled by default in production mode.

If you still feel like adding it here, please use the same phrasing as we have:

```
By default this plugin is already enabled in production [`mode`](/concepts/mode/#mode-production) and disabled elsewise.
```

Reply:
Done